### PR TITLE
fix(dropdown): Fix ellipsis not to working for simple text dropdown

### PR DESCRIPTION
**Pull request by rivka-ungar**
https://monday.monday.com/boards/3532714909/views/80492480/pulses/7031893214

### CODE REVIEW

**Review by talkor — APPROVED** [review]
LGTM
@YossiSaadi WDYT?

File: packages/core/src/components/Dropdown/components/ChildrenContent/ChildrenContent.jsx [hunk 1]
```diff
@@ -36,7 +37,9 @@ export const ChildrenContent = ({ data, children, readOnly }) => {
           {data?.leftIcon && <Icon iconSize={18} icon={data.leftIcon} clickable={false} />}
         </>
       )}
-      {children}
+      <Text type="text2" color="inherit">
```
Comment (YossiSaadi) [inline packages/core/src/components/Dropdown/components/ChildrenContent/ChildrenContent.jsx:40]: Is it working and not throwing errors? we've only allowed this at the Vibe3 branch, didn't we?

**Review by YossiSaadi — APPROVED** [review]
Looks cool, had one concern

File: packages/core/src/components/Dropdown/components/ChildrenContent/ChildrenContent.jsx (hunk 1 above)
Comment (rivka-ungar) [reply packages/core/src/components/Dropdown/components/ChildrenContent/ChildrenContent.jsx]: Weird that it didn't throw any errors, changed it

**Pull request merged**